src/main/java/xbot/common/controls/actuators/wpi_adapters/CANSparkMaxWpiAdapter.java

```diff
@@ -258,19 +259,21 @@ public boolean getFault(FaultID faultID) {
         return internalSpark.getFault(faultID);
     }
 
-    public boolean getStickyFault(FaultID faultID) {
+    public boolean getStickyFault(FaultID faultID) { return internalSpark.getStickyFault(faultID); }
+
+    public boolean getStickyFault_internal(FaultID faultID) {
         return internalSpark.getStickyFault(faultID);
     }
 
-    public double getBusVoltage() {
+    public double getBusVoltage_internal() {
```

Comment:
is this underscore naming coming from the can spark max interface?

Reply:
No - this is me trying to keep most things backwards compatible, but a better name would be welcome. The basic idea is as such:

* The `XCANSparkMax` abstract class exposes the "simple" name like `getPosition()`, since that's what subsystems/etc. will be calling.
* The implementations like `CANSparkMaxWpiAdapter` need to immediately return the info, but per AdvantageKit philosophy, they can't contact the underlying object; they instead need to refer to a cached value. So `getPosition()` just calls `return inputs.position`.
* At some point we do actually need to call the underlying object to fill that cache, and that method is `getPosition_internal()`, called in `updateInputs()`